IBPSA/ThermalZones/ISO13790/Zone5R1C/Zone.mo

```diff
@@ -0,0 +1,327 @@
+      extent={{-6,3},{-6,3}},
+      horizontalAlignment=TextAlignment.Right));
+  connect(solGai.u1, glaEle.solRadWin) annotation (Line(points={{-62,-64},{-72,
+          -64},{-72,-50},{-79.2857,-50}}, color={0,0,127}));
+  connect(solGai.u2, opaEle.SolRadOpa) annotation (Line(points={{-62,-76},{-72,
+          -76},{-72,-89.5238},{-79.5455,-89.5238}}, color={0,0,127}));
+  connect(phiSur.solGai, solGai.y) annotation (Line(points={{122,-8},{128,-8},{
+          128,-102},{-30,-102},{-30,-70},{-39,-70}}, color={0,0,127}));
+  connect(phiMas.solGai, solGai.y) annotation (Line(points={{122,-88},{128,-88},
+          {128,-102},{-30,-102},{-30,-70},{-39,-70}}, color={0,0,127}));
+  connect(phiAir.y, heaAir.Q_flow)
+    annotation (Line(points={{99,80},{80,80}}, color={0,0,127}));
+  connect(phiSur.surGai, heaSur.Q_flow)
+    annotation (Line(points={{99,0},{80,0}}, color={0,0,127}));
+  connect(phiMas.masGai, heaMas.Q_flow)
+    annotation (Line(points={{99,-80},{80,-80}}, color={0,0,127}));
+  connect(phiAir.u, intGai) annotation (Line(points={{122,80},{134,80},{134,
+          -134},{-70,-134},{-70,-120},{-160,-120}}, color={0,0,127}));
+  connect(phiSur.intGai, intGai) annotation (Line(points={{122,-4},{134,-4},{
+          134,-134},{-70,-134},{-70,-120},{-160,-120}}, color={0,0,127}));
+  connect(phiMas.intGai, intGai) annotation (Line(points={{122,-84},{134,-84},{
+          134,-134},{-70,-134},{-70,-120},{-160,-120}}, color={0,0,127}));
+  annotation (defaultComponentName="zon5R1C", Icon(coordinateSystem(preserveAspectRatio=false, extent={{-140,
+            -140},{140,140}}), graphics={Rectangle(
+          extent={{-140,140},{140,-140}},
+          lineThickness=1,
+          pattern=LinePattern.None,
+          lineColor={0,0,0},
+          fillColor={100,0,0},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-120,120},{120,-120}},
+          lineColor={28,108,200},
+          fillColor={170,213,255},
+          fillPattern=FillPattern.Sphere),
+        Text(
+          extent={{-104,174},{118,142}},
+          textColor={0,0,255},
+          textString="%name")}),Diagram(coordinateSystem(preserveAspectRatio=
+            false, extent={{-140,-140},{140,140}})),
+    Documentation(info="<html>
+<p>
+This is a lumped-capacitance simplified building model based  on the 5R1C 
+network presented in the ISO 13790:2008 Standard. The simplified 5R1C model uses
+five thermal resistances and one thermal capacitance to reproduce the 
+transient thermal behaviour of buildings. The thermal zone is modeled with three 
+temperature nodes, the indoor air temperature <code>Tair</code>, the envelope internal 
+surface temperature <code>Tsur</code> and the building mass temperature Tmas (heat port neglected), and two boundary
+condition nodes, supply air temperature <code>Tsup</code> and the external air temperature 
+<code>Text</code>. The five resistances are related to heat transfer by ventilation <code>Hve</code>, 
```

Comment:
Use either code as you have now, but then use the right capitalization as in <code>TAir</code>, or use <i>T<sub>air</sub></i>.

Reply:
Done